File: business/src/main/java/no/nav/sbl/sosialhjelp/pdfmedpdfbox/PdfGeneratorConfig.java
package no.nav.sbl.sosialhjelp.pdfmedpdfbox;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import javax.xml.soap.Text;
import java.io.IOException;

@Configuration
public class PdfGeneratorConfig {

    @Bean
    public PdfGenerator pdfGenerator() throws IOException {
        return new PdfGenerator();
    }

    @Bean
    public SosialhjelpPdfGenerator sosialhjelpPdfGenerator() {
        return new SosialhjelpPdfGenerator();
    }

    @Bean
    public TextHelpers textHelpers() {
        return new TextHelpers();
    }

}

File: business/src/main/kotlin/no/nav/sbl/sosialhjelp/pdfmedpdfbox/PdfGeneratorConfig.kt
package no.nav.sbl.sosialhjelp.pdfmedpdfbox

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.io.IOException

@Configuration
open class PdfGeneratorConfig {
    @Bean
    @Throws(IOException::class)
    open fun pdfGenerator(): PdfGenerator {
        return PdfGenerator()
    }

    @Bean
    open fun sosialhjelpPdfGenerator(): SosialhjelpPdfGenerator {
        return SosialhjelpPdfGenerator()
    }

    @Bean
    open fun textHelpers(): TextHelpers {
        return TextHelpers()
    }
}